examples/distributed/distributed_debate.py

```diff
@@ -65,39 +65,44 @@ def setup_server(parsed_args: argparse.Namespace) -> None:
         encoding="utf-8",
     ) as f:
         configs = json.load(f)
+        configs = {
+            "pro": configs[0]["args"],
+            "con": configs[1]["args"],
+            "judge": configs[2]["args"],
+        }
         config = configs[parsed_args.role]
         host = getattr(parsed_args, f"{parsed_args.role}_host")
         port = getattr(parsed_args, f"{parsed_args.role}_port")
         server_launcher = RpcAgentServerLauncher(
+            agent_class=DialogAgent,
+            agent_kwargs=config,
             host=host,
             port=port,
-            local_mode=False,
-            agent_class=RpcDialogAgent,
-            **config,
         )
         server_launcher.launch()
         server_launcher.wait_until_terminate()
 
 
 def run_main_process(parsed_args: argparse.Namespace) -> None:
     """Setup the main debate competition process"""
-    agentscope.init(
+    agents = agentscope.init(
```

Comment:
`pro_agent, con_agemt, judge_agent = agentscope.init ...` may be better instead of codes in lines 92-94.